examples/VREP/orientation_control_osc.py

```diff
@@ -0,0 +1,70 @@
+"""
+Running operational space control using VREP. The controller will
+move the end-effector to the target object's orientation.
+"""
+import numpy as np
+import pygame
+
+from abr_control.arms import ur5 as arm
+from abr_control.controllers import OSC
+from abr_control.interfaces import VREP
+from abr_control.utils import transformations
+
+
+# initialize our robot config
+robot_config = arm.Config(use_cython=True)
+# create opreational space controller
+ctrlr = OSC(robot_config, kp=50)
+
+# create our interface
+interface = VREP(robot_config, dt=.005)
+interface.connect()
+
+# set up lists for tracking data
+ee_track = []
+
+# control (alpha, beta, gamma) out of [x, y, z, alpha, beta, gamma]
+ctrlr_dof = [False, False, False, True, True, True]
+
+
+try:
+    count = 0
+    print('\nSimulation starting...\n')
+    while 1:
+        # get arm feedback
+        feedback = interface.get_feedback()
+        hand_xyz = robot_config.Tx('EE', feedback['q'])
+
+        target = np.hstack([
+            interface.get_xyz('target'),
+            interface.get_orientation('target')])
+
+        u = ctrlr.generate(
+            q=feedback['q'],
```

Comment:
is this just setting the target pos and orientation to the what the target is? When I run the sim I don't see it reaching to anything, and the ball labelled 'target' is below the floor surface.

Reply:
oh right, need to change the default position of target haha, and yes that's what it's doing